samples/go/photo-index/main.go

```diff
@@ -171,11 +187,13 @@ func index() (win bool) {
 	}
 
 	outDS, err = db.Commit(outDS, types.NewStruct("", types.StructData{
-		"byDate":       byDate.Build(),
-		"byTag":        byTag.Build(),
-		"byFace":       byFace.Build(),
-		"tagsByCount":  stringsByCount(db, tagCounts),
-		"facesByCount": stringsByCount(db, faceCounts),
+		"byDate":         byDate.Build(),
+		"byFace":         byFace.Build(),
+		"bySource":       bySource.Build(),
+		"byTag":          byTag.Build(),
+		"facesByCount":   stringsByCount(db, faceCounts),
+		"sourcesByCount": stringsByCount(db, sourceCounts),
+		"tagsByCount":    stringsByCount(db, tagCounts),
```

Comment:
Will you want to get the N most common tags for any reason? In which case you should add a "sourcesByCount" here as well.